examples/bento/no-v0.html

```diff
@@ -0,0 +1,315 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>bento-timeago</title>
+    <link rel="canonical" href="amps.html" />
+    <meta
+      name="viewport"
+      content="width=device-width,minimum-scale=1,initial-scale=1"
+    />
+    <style data-bento-boilerplate>
+      bento-timeago {
+        display: inline-block;
+        position: relative;
+        overflow: hidden;
+      }
+    </style>
```

Comment:
Is this preferred over including `<link rel="stylesheet" href="https://cdn.ampproject.org/v0/amp-timeago-1.0.css">`?

Reply:
As initial style yes, to prevent CLS.